airbyte-ci/connectors/pipelines/pipelines/sentry_utils.py

```diff
@@ -0,0 +1,40 @@
+import sentry_sdk
+from connector_ops.utils import Connector
+
+
+def with_step_context(func):
+    def wrapper(self, *args, **kwargs):
+        with sentry_sdk.configure_scope() as scope:
+            step_name = self.__class__.__name__
+            scope.set_tag("pipeline_step", step_name)
+            scope.set_context("Pipeline Step", {
+                'name': step_name,
+                'step_title': self.title,
+                'max_retries': self.max_retries,
+                'retry_count': self.retry_count,
+            })
+
+            if hasattr(self.context, "connector"):
+                connector: Connector = self.context.connector
+                scope.set_tag("connector", connector.technical_name)
+                scope.set_context("Connector", {
+                    'name': connector.name,
+                    'technical_name': connector.technical_name,
+                    'language': connector.language,
```

Comment:
Shall we add the version + release stage?
```suggestion
                    'language': connector.language,
                    'version': connector.version
                    'release_stage': connector.release_stage
```